packages/opencensus-exporter-jaeger/src/jaeger-driver/index.ts

```diff
@@ -59,13 +59,16 @@ export function spanToThrift(span: Span) {
 
   const spanLogs = [];
   const unsigned = true;
-  const length = span.spanContext.traceId.length;
   const parentSpan = span.parentSpanId ? Utils.encodeInt64(span.parentSpanId) :
                                          ThriftUtils.emptyBuffer;
 
+  const numericTraceId = Number('0x' + span.spanContext.traceId);
+  const traceIdLow = numericTraceId & (2 ^ 64 - 1);
```

Comment:
Evaluating in an interpreter, `2 ^ 64 - 1` yields `61`. I get a feeling that the operation here won't work for JavaScript. Maybe we should be using Buffers here instead to manipulate byte arrays?

Reply:
@kjin sorry, I raised a couple of these a bit prematurely. I need to add tests or this and actally try it out.